src/biokbase/narrative/tests/test_jobmanager.py

```diff
@@ -36,6 +37,7 @@
     BATCH_CHILDREN,
     BATCH_ERROR_RETRIED,
     BATCH_PARENT,
+    BATCH_RETRY_RUNNING,
```

Comment:
is this used in this file? if not, can you remove it from the imports?

The VSCode python syntax highlighting / checking should indicate where you have unused imports.

Reply:
Oops yeah I should run flake8

Reply:
You can enable flake8 linting directly in vscode if you turn on / configure the various python extensions, so you can catch these things as you're editing. 🎉

Reply:
Heh yeah I have that ... you should see my code from when I used vim